packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -59,7 +47,7 @@ export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
           role={'columnheader'}
           aria-sort={column.isSorted ? (column.isSortedDescending ? 'descending' : 'ascending') : 'none'}
           aria-colindex={columnIndex}
-          className={classNames.root}
+          className={this._classNames.root}
```

Comment:
Locals like this are used to get the minified size down and shouldn't be removed (at least in cases like this where it's used several times). It's obviously a small impact per instance, but helps fight "death by a thousand paper cuts" overall.

Reply:
>Locals like this are used to get the minified size down and shouldn't be removed (at least in cases like this where it's used several times).

@ecraig12345 before we optimize, what is the un/minified delta?

>It's obviously a small impact per instance, but helps fight "death by a thousand paper cuts" overall.

As I'm in the process of fighting "death by a thousand paper cuts" where folks unknowingly regressed React reconciliation across `DetailsList` (see https://github.com/OfficeDev/office-ui-fabric-react/pull/8799), I see having a single source of truth as preferable. Especially when dealing with `styles` and the like.

Also, the overall minification size was _reduced_ as part of this PR (https://github.com/OfficeDev/office-ui-fabric-react/pull/8872#issuecomment-487702603).

Reply:
I will try to pull the minification delta out of curiosity. If the delta is non-negligible then I will revert those LoC.

Have a few pressing things to attend to first.

Reply:
Basically yeah. You add a bit for `var a=this._classNames;` (or similar) but get 12x `a.foo` instead of 12x `this._classNames.foo`. Extracting things into locals has become pretty common practice in our codebase for that reason (and in ODSP, which was how I learned about this), though I'm not sure if anyone has calculated the total impact.

I can un-block the PR if you want, but I wanted to at least call out that part of what you're removing was added for a reason.

Reply:
Let's have `size-auditor` run with the updated commits. I'd like to compare the delta for my own intellectual curiosity.

Reply:
`-170` byte delta, seems worthwhile to pursue. Thanks for helping to save additional bytes across the wire.

Would be great to one day have a holistic solution for the repo to minimize asset footprint.